### PR TITLE
feat(account-lib): allow creating ETH Keypair from provided or random…

### PR DESCRIPTION
**Pull request by fmartinAltamirano**
Currently, the constructor of the KeyPair class for eth doesn’t allow creating an instance from a provided or random seed.
This change will be required in order to make the abstractEthLikeCoin in the SDK use account-lib for key pair generation.

Link to JIRA:[ STLX-5694](https://bitgoinc.atlassian.net/browse/STLX-5694)

### CODE REVIEW

**Review by smaria95 — APPROVED** [review]
LGTM

File: modules/account-lib/src/coin/eth/keyPair.ts [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import { HDNode } from '@bitgo/utxo-lib';
 import { addHexPrefix, pubToAddress } from 'ethereumjs-util';
-import { DefaultKeys, isPrivateKey, isPublicKey, KeyPairOptions } from '../baseCoin/iface';
+import { DefaultKeys, isPrivateKey, isPublicKey, isSeed, KeyPairOptions } from '../baseCoin/iface';
 import { Secp256k1ExtendedKeyPair } from '../baseCoin/secp256k1ExtendedKeyPair';
+import { randomBytes } from 'crypto';
+
+const DEFAULT_SEED_SIZE_BYTES = 64;
```
[the remark this answers is not in the document]
Comment (fmartinAltamirano) [reply modules/account-lib/src/coin/eth/keyPair.ts:7]: Hi @CapnMigraine , for the particular case of Avaxc we used the value of other coins (use secp256k1), in ETH I took [the size used in the SDK](https://github.com/BitGo/BitGoJS/blob/master/modules/core/src/v2/coins/eth.ts#L1604-L1609). 
Is it ok this approach for you?

**Review by CapnMigraine — DISMISSED** [review]
lgtm

File: modules/account-lib/src/coin/eth/keyPair.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (CapnMigraine) [reply modules/account-lib/src/coin/eth/keyPair.ts:7]: Yes, we should probably use 64 bytes in AVAX too

**Review by CapnMigraine — APPROVED** [review]

**Pull request merged**